charts/controller/pre-job-hook.sh

```diff
@@ -0,0 +1,35 @@
+# Install the Semaphore toolbox in the job
+rm -rf ~/.toolbox
+
+downloadPath="https://github.com/semaphoreci/toolbox/releases/download/v1.19.40/self-hosted-linux.tar"
```

Comment:
@lucaspin is it possible to parametrize toolbox version and have fallback to the default